messaging/WFLY-13991_keep_corrupted_journal_files.adoc

```diff
@@ -0,0 +1,66 @@
+= Keep corrupted Journal filesin Artemis.
+:author:            Emmanuel Hugonnet
+:email:             your.email@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+
+When a corrupted artemis journal file is seen Artemis shuts down EAP, but before it does that it deletes the corrupted file. This means that no after the event analysis/investigation can be done as the corrupted journal file is gone.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13991[WFLY-13991]
+
+=== Related Issues
+
+* https://issues.apache.org/jira/browse/ARTEMIS-2939[ARTEMIS-2939]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+
+* [X] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* WildFly
+* ActiveMQ Artemis (finished)
+
+=== Other Interested Projects
+
+== Requirements
+
+* The journalMaxAtticFiles parameter of ActiveMQ Artemis configuration must be configurable via Wildfly management model.
+
+== Implementation Plan
+
+The `journal-max-attic-files` attribute of type INT with a default value of '10' will be added to messaging server resource.
```

Comment:
Is it possible to set infinite here? Eg by setting `0` value?

Reply:
No
0 means no file stored